ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt

```diff
@@ -21,14 +12,23 @@ if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
         image_transport
         camera_info_manager
         nodelet
-    )
+        )
 
-    catkin_package(
+catkin_package(
```

Comment:
This will declare a new exported library, but it won't exist unless we're building for the DrivePX

Reply:
I see. I agree with that option.